guides/source/upgrading_ruby_on_rails.md

```diff
@@ -1725,7 +1725,7 @@ Rails 4.0 no longer supports loading plugins from `vendor/plugins`. You must rep
 * Rails 4.0 has changed `serialized_attributes` and `attr_readonly` to class methods only. You shouldn't use instance methods since it's now deprecated. You should change them to use class methods, e.g. `self.serialized_attributes` to `self.class.serialized_attributes`.
 
 * When using the default coder, assigning `nil` to a serialized attribute will save it
-to the database as `NULL` instead of passing the `nil` value through YAML (`"--- \n...\n"`).
+to the database as `NULL`. Previously, if the database column corresponding to the serialized attribute was nullable, the `nil` value through YAML was saved into the database as `"--- \n...\n"`. ActiveRecord was, and still will, return `nil` when accessing the serialized attribute. However, If the database column was not nullable, the `nil` value through YAML was saved into the database as `"--- \'\'\n"`, representing an empty string. Previously, ActiveRecord would return an empty string when accessing the serialized attribute. In this case, you will need to change the database column to allow `NULL` and ActiveRecord will now return `nil` when accessing the serialized attribute. Be aware of any code you might have that assumes the serialized attribute will not be `nil` in that case. 
```

Comment:
This is confusing. It is saying `Previously` twice so I could not understand what was the old behavior and what is the new behavior. I think we would be better served with an example.

Also `ActiveRecord` should be changed with `Active Record`.

Reply:
@rafaelfranca Thanks for the feedback. I made some changes.